src/screens/ReportScreen.js

```diff
@@ -1,24 +1,179 @@
-       * we just wanted to provide you with some helpful links.
-       */}
-      <ExpoLinksView />
-    </ScrollView>
+    <LinearGradient colors={["#000", "#000"]} style={styles.container}>
+      <Text style={styles.title}>Since the exposure, have you felt any of the following?</Text>
+      <View style={styles.switchContainer}>
+        <View style={styles.row}>
+          <Switch
+            trackColor={TRACK_COLOR}
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "lossOfSenseOfSmell": v})}
+            value={symptoms["lossOfSenseOfSmell"]}
+          />
+          <Text style={styles.label}>
+            Loss of Sense of Smell
+          </Text>
+        </View>
+        <View style={styles.row}>
+          <Switch
+            trackColor={TRACK_COLOR}
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "cough": v})}
+            value={symptoms["cough"]}
+          />
+          <Text style={styles.label}>
+            Cough
+          </Text>
+        </View>
+        <View style={styles.row}>
+          <Switch
+            trackColor={TRACK_COLOR}
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "fever": v})}
+            value={symptoms["fever"]}
+          />
+          <Text style={styles.label}>
+            Fever
+          </Text>
+        </View>
+        <View style={styles.row}>
+          <Switch
+            trackColor={TRACK_COLOR}
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "tiredness": v})}
+            value={symptoms["tiredness"]}
+          />
+          <Text style={styles.label}>
+            Tiredness
+          </Text>
+        </View>
+        <View style={styles.row}>
+          <Switch
+            trackColor={TRACK_COLOR}
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "difficultyBreathing": v})}
+            value={symptoms["difficultyBreathing"]}
+          />
+          <Text style={styles.label}>
+            Difficulty Breathing
+          </Text>
+        </View>
+        <View style={styles.row}>
+          <Switch
+            trackColor={TRACK_COLOR}
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "pressureInChest": v})}
+            value={symptoms["pressureInChest"]}
+          />
+          <Text style={styles.label}>
+            Pressure in Chest
+          </Text>
+        </View>
+        <View style={styles.row}>
```

Comment:
can DRY this up put these Views into functions or new components, for example

```
const symptomRow = ({symptoms, type, description}) => {
  return (
    <View style={styles.row}>
      <Switch
        trackColor={TRACK_COLOR}
        thumbColor={THUMB_COLOR}
        ios_backgroundColor={IOS_SWITCH_BG_COLOR}
        onValueChange={(v) => setSymptoms({...symptoms, type: v})}
        value={symptoms[type]}
      />
      <Text style={styles.label}>
        {description}
      </Text>
    </View>
  )
}
```

then you can invoke from the view
```
{ symptomRow({ symptoms, type: "pressureInChest", description: "Pressure in Chest"}) }
{ symptomRow({ symptoms, type: "difficultyBreathing", description: "Difficulty Breathing"}) }
```
or evein in a loop

Reply:
That's fair!